docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,259 @@
+
+#### Defining params
+
+In the v1 workflow builder, params could be structured or unstructured. The [params schema builder](https://pipedream.com/docs/v1/workflows/steps/params/#configuring-custom-params) allowed you to add your own custom params to steps.
+
+In v2, you can add your own custom props without leaving the code editor with `props`.
+
+```javascript
+export default defineComponent({
+  props: {
+    firstName: {
+      type: 'string',
+      label: 'Your first name',
+    }
+  },
+  async run({ steps, $ }) {
+    console.log(this.firstName);
+  }
+});
+
+```
+
+In the example above a `firstName` string prop is created. The value assigned to this prop in the workflow builder by either a static string or dynamic data will be available in the step as `this.firstName`.
+
+Additionally, a visual component is rendered in the step **Configuration** to accept this input:
+
+<img src="./images/custom-string-prop.png" alt="Custom props render in the Configuration portion of the code step.">
+
+### Connecting apps
+
+In the v2 builder, you can connect apps with your code using [props](/components/props).
+
+Above the `run` function define an app prop that your Node.js step integrates with.
+
+```javascript
+export default defineComponent({
+  props: {
+    twitter: {
+      type: "app",
+      app: "slack",
+    }
+  },
+  async run({ steps, $ }) {
+    // now your Slack token for authenticating API calls will be available here:
+    this.slack.$auth.oauth_access_token;
+  },
+})
+```
+
+After testing the step the Slack app will appear in the **Configuration** section on the left hand side. In this section you can choose which Slack account you'd like to use in the step.
+
+<img src="./images/app-props-example.png" alt="Example of adding an app connection to a v2 Node.js step">
+
+
+### HTTPs Response
+
+You can still return an HTTP response from an HTTP triggered workflow.
+
+Use `$.respond` to send a JSON or string response from the HTTP call that triggered the workflow.
+
+```javascript
+export default defineComponent({
+  async run({ steps, $ }) {
+    $.respond({
+      status: 200,
+      headers: {},
+      body: { 
+        message: "hello world!"
+      }
+    });
+  },
+})
+```
+
+Please note, you'll also need to configure the HTTP trigger step to also allow custom responses. Use the dropdown in the **HTTP Response** section of the HTTP trigger to select the **Return a custom response from your workflow** option.
+
+<img src="./images/custom-http-response-option.png" alt="Select the option to allow your workflow to send it's own HTTP responses">
+
+## Known Gaps & Limitations
+
+The new workflow builder introduces new features that weren't available in v1. However there are some features from the original builder that are not available in v2 currently.
```

Comment:
"However, some features from the original builder are not currently available in v2. The Pipedream team is working to quickly address these items, but if you have feedback that isn't listed here, please [reach out](https://pipedream.com/support)".